AusDTO/dto-digitalmarketplace-buyer-frontend
# Fix integration tests

- install latest firefox on circle
- install geckdriver on circle
- after each button/link click, add checks for text on next page to ensure next page has loaded
- add hacky code to handle random StaleElementReferenceException's from selenium
